scripts/convert.js

```diff
@@ -21,11 +21,12 @@ const indicators = Object.entries(INDICATORS).map(([key, indicator]) => ({
 getStdin().then((rawCsv) => {
   csv.parse(rawCsv, { columns: true }, (err, data) => {
     const cities = data.map((row) => {
-      const city = {
-        name: row.Cities,
-        id: kebabCase(row.Cities),
-        indices: {},
-      };
+      const city = CITIES[row.Cities];
+
+      if (!city) throw new Error(`${row.Cities} is not a known smart city`);
+
+      city.name = row.Cities;
+      city.indices = {};
```

Comment:
We now mix in some static city data in the same way we get the static indicator data.